### PR TITLE
Improve CONTRIBUTING doc

### PR DESCRIPTION
**Pull request by konekoya**
Hi Allen,

Sorry for this super late PR. Life has been very busy these days.
I've made several tweaks and hoping that I didn't change the original meaning that you're
trying to express in the doc.

Cheers!

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,21 +1,22 @@
...
+Anyway, you're always welcome to open an issue to ask questions or talk about new feature request.
 
 # Pull Requests
 Check [here](./docs/development.md) for getting started with development.
 
-* When you want to implement a new feature, please let us know (via issues).
-* Please run tests before opening a PR and also remember to test the actual business logic.
-* If your PR is trying to fix a bug, please describe the bug number with hashtag when creating the PR.
+* We recommend filing an [issue](https://github.com/react-bootstrap-table/react-bootstrap-table2/issues) before you implement any new features.
+* Please ensure all the test suit passes before submitting a PR. And always test the actual business logic.
```
Comment (Chun-MingChen) [inline CONTRIBUTING.md]: Please ensure that all the test suites have passed before submitting a PR. Besides, always test the actual business logic.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,21 +1,22 @@
...
+* If your PR is for fixing bug, please describe the bug when submitting the PR. and if possible, including the bug number with hastag.
```
Comment (Chun-MingChen) [inline CONTRIBUTING.md]: If your PR is trying to fix the bug, please describe the details as much as you could and tag bug number with `hashtag`.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,21 +1,22 @@
...
+Before opening an issue, please make sure your problem or request doesn't exist. When opening an issue, please provide the following information if possible:
 
 * Example code or repo (please keep it simple and minimal)
-* Steps to reproduce
-* `react-bootstrap-table2` version
+* Steps to reproduce.
+* `react-bootstrap-table2` version.
 
-Anyway, you're welcome to open an issue to ask questions or discuss about a feature request.
+Anyway, you're always welcome to open an issue to ask questions or talk about new feature request.
```
Comment (Chun-MingChen) [inline CONTRIBUTING.md]: Additionally, questions asking and new feature requests are welcomed through the issue.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,21 +1,22 @@
...
+* If your PR is for fixing bug, please describe the bug when submitting the PR. and if possible, including the bug number with hastag.
 
 # For the members of react-bootstrap-table2 org
-
-* Please convert the ticket to issue when is ticket moved from `Backlog` to `Ready` in project.
-* Please write documentation if any new API/feature/props is changed or added.
-* Please add a story example if any new feature is implemented
\ No newline at end of file
+* Please convert the ticket to issue when the ticket has moved from `Backlog` to `Ready`.
+* Please update the docs with any API, feature, component props changes or added, the code and docs should always be in sync.
```
Comment (Chun-MingChen) [inline CONTRIBUTING.md]: Please update the docs if any API, feature or component props was changed or added. The code and docs should always be in sync.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,21 +1,22 @@
...
+* Please add a story example if any new feature is implemented.
```
Comment (Chun-MingChen) [inline CONTRIBUTING.md]: Please add a story example if any new feature was implemented.

Comment (konekoya) [reply CONTRIBUTING.md]: Ah, good catch! Yes, it should be "was" not "is".

File: CONTRIBUTING.md (hunk 1 above)
Comment (konekoya) [reply CONTRIBUTING.md]: Yeah, this looks better :)

File: CONTRIBUTING.md (hunk 2 above)
Comment (konekoya) [reply CONTRIBUTING.md]: This looks good, but a small change i would make:
If your PR is trying to fix "a" bug not "the" bug.

File: CONTRIBUTING.md (hunk 4 above)
Comment (konekoya) [reply CONTRIBUTING.md]: Thank you, didn't notice I was using different tenses in a sentence...

File: CONTRIBUTING.md (hunk 3 above)
Comment (konekoya) [reply CONTRIBUTING.md]: I'm not sure about the usage "questions asking"... What about:
Additionally, asking questions and requesting new features are welcomed via [issues](https://github.com/react-bootstrap-table/react-bootstrap-table2/issues)

**Comment by AllenFang** [comment]
@konekoya @Chun-MingChen thanks you guys contributions, is there any changes @konekoya you need to commit? if yes, please remember to rebase, thanks :)

File: CONTRIBUTING.md (hunk 3 above)
Comment (Chun-MingChen) [reply CONTRIBUTING.md]: Actually, `questions asking` and `new feature requests` are nouns which emphasize these behavior. However, I thought both of that are easy to understand. Therefore, it's up to you :)

Comment (konekoya) [reply CONTRIBUTING.md]: Okay, I see your point here. Thank you 💯

**Comment by konekoya** [comment]
@AllenFang @Chun-MingChen 
I have finished rebasing and added a new commit for what @Chun-MingChen has suggested above. Please review it.
Thank you :)

**Comment by AllenFang** [comment]
@konekoya thanks :)

**Pull request merged**